eng/jacoco-test-coverage/pom.xml

```diff
@@ -88,6 +89,11 @@
       <artifactId>azure-messaging-eventhubs</artifactId>
       <version>${azure-messaging-eventhubs.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-storage-queue</artifactId>
+      <version>${azure-storage-queue.version}</version>
+    </dependency>
```

Comment:
Alphabetical here (based on artifactId) - push to after blob below.

Reply:
Done.